### DISCUSSION TITLE
Advanced Settings - isFTXM

### DISCUSSION THREAD

**Discussion by Mark-HomeAssistantTC** [revk/xESP32-Faikout#555] [Q&A] [answered]
Hello,

what does the setting "isFTXM - Use control for FTXM Perfera Units (Experimental)" do?
I can't See any differences switch this setting on or off.
Use a Daikin FTXM35A.

Kind regards

**Comment by revk** [comment, marked as answer]
Experimental usually means it does not work yet, may not even be coded yet.
